pygmt/helpers/decorators.py

```diff
@@ -82,6 +82,13 @@
             ``[g|p]``
             Force output grid to be gridline (g) or pixel (p) node registered.
             Default is gridline (g).""",
+    "t": """\
+        transparency : float
```

Comment:
`transparency` can also be an array for a few modules, e.g., `plot`, `plot3d` and `text`, so that symbols and texts can have varying transparency. It needs more work and should be addressed in separated PRs.

Reply:
Ok, but could you add at least one unit test for `transparency` on a module that won't have array-like `transparency` inputs (e.g. `image`)?

Reply:
It means adding another baseline image to the repository. 😄

Reply:
Maybe just a gallery example then, if not too much work :no_mouth: I suppose we could come up with a smart `check_figures_equal` test but happy to leave that to the next PR.

Reply:
There is an upcoming example in https://github.com/GenericMappingTools/pygmt/pull/607, but I'm not sure if the example does too many things (symbols, transparency and legends.).

Reply:
Ah right, missed the transparency part on that example. I don't think we can have too many examples, if anything we have too little right now (a lot of catching up to do with matplotlib at https://matplotlib.org/gallery/index.html)!